test/razorpay/test_order.rb

```diff
@@ -44,5 +44,27 @@ def test_order_payments_should_be_fetched
       assert !payments.items.empty?, 'payments should be more than one'
       assert_equal 'pay_50sbkZA9AcyE5a', payments.items[0]['id'], 'payment id should match'
     end
+
+    def test_edit_order
+      param_attr = {
+        "notes": {
+          "key1": "value3",
+          "key2": "value2"
+        }
+      }
+     
+      stub_patch(%r{orders/#{@order_id}$}, 'fake_order', param_attr.to_json)
+      order = Razorpay::Order.edit(@order_id, param_attr.to_json)
+      assert_instance_of Razorpay::Order, order, 'order not an instance of Razorpay::Order class'
+      assert_equal @order_id, order.id, 'order IDs do not match'
+   end
+
+   def test_fetch_order_transfers
+    stub_get("#{BASE_URI}orders/#{@order_id}/?expand[]=transfers&status", 'fake_order_transfers')
+    order = Razorpay::Order.fetchTransferOrder(@order_id)
+    assert_instance_of Razorpay::Order, order, 'order not an instance of Razorpay::Order class'
+    assert_equal @order_id, order.id, 'order IDs do not match'
+    refute_empty order.transfers
```

Comment:
check `order.transfers.items`. `transfers` has other attr which is not indicative of actual item collection

Reply:
i have checked order.transfers.items and added assertion for transfers.items